extensions.json

```
@@ -688,6 +688,11 @@
       "id": "zxh404.vscode-proto3",
       "repository": "https://github.com/zxh0/vscode-proto3",
       "version": "0.4.2"
+    },
+    {
+      "id": "platformio.platformio-ide",
+      "repository": "https://github.com/platformio/platformio-vscode-ide",
+      "version": "1.10.0"
```

When specifying a `version`, it's good to also specify a corresponding `checkout` (that way we can be certain that the version we publish is in the state that the maintainer intended it to be, as opposed to publishing it once from `master`)

```suggestion
      "version": "1.10.0",
      "checkout": "v1.10.0"
```